src/app/learnocaml_partition_view.ml

```diff
@@ -34,10 +33,27 @@ let open_tok tok =
   let _win = window_open ("/student-view.html?token="^tok) "_blank" in
   false
 
+(* let list_of_nick = 
+  List.map @@ fun x ->
+    let tok = Token.to_string x in
+    let nick = "toto" in (*trouver comment récupérer les pseudos*)
```

Comment:
With the currently implemented Web API (`src/state/learnocaml_api.ml`), the nickname data can be retrived from `retrieve (Learnocaml_api.Students_list token)`

Actually, this is already done in `learnocaml_teacher_tab.ml` and stored in this variable:

https://github.com/ocaml-sf/learn-ocaml/blob/8878c9b5557fb8ddc76aa3b68d03a25f06983a2d/src/app/learnocaml_teacher_tab.ml#L113

https://github.com/ocaml-sf/learn-ocaml/blob/8878c9b5557fb8ddc76aa3b68d03a25f06983a2d/src/app/learnocaml_teacher_tab.ml#L1341-L1345

Reply:
Concretely, it has type:

```
students_map : Learnocaml_data.Student.t Learnocaml_data.Token.Map.t ref
```

so it is possible to get the `value : Student.t` corresponding to the desired students, and get their nickname:

https://github.com/ocaml-sf/learn-ocaml/blob/8878c9b5557fb8ddc76aa3b68d03a25f06983a2d/src/state/learnocaml_data.mli#L100-L108

Reply:
thanks, i'm woking on it